estate/views/estate_searchs.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="estate_search" model="ir.ui.view">
+        <field name="name">estate.property.search</field>
```

Comment:
Guildines : `id="estate_property_view_search"` = 'modelName_view_viewType'
and the name should be the same but using '.' instead of '_'
apply this everywhere :)